tests/foreman/api/test_organization.py

```diff
@@ -77,7 +78,10 @@ def test_positive_create(self):
             headers={'content-type': 'text/plain'},
```

Comment:
For workaround here, wouldn't it better to change content-type instead when bz is open with `is_open` and not checking UNSUPPORTED_MEDIA_TYPE, wdyt?

Reply:
As it is now, we're testing that UNSUPPORTED_MEDIA_TYPE is being returned, or at least ISE 500 if the BZ is open (as opposed to returning something even more wrong, e.g. 200 OK).

Reply:
IMO using any other content-type's isn't wrong, and I feel its better option to test something than checking if we hit UNSUPPORTED_MEDIA_TYPE/ISE :)

Its upto you how you want to test it, so its non-blocking.